examples/SDL2/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include<iostream>
+#include<SDL2/SDL.h>
+
+int main(int argc, char *argv){
```

Comment:
> No, just a simple example

Did it work? because according to the docs, SDLmain is required on MSVC, WinRT, Android?, and some more